packages/react-dom/src/__tests__/ReactBrowserEventEmitter-test.js

```diff
@@ -347,14 +365,4 @@ describe('ReactBrowserEventEmitter', () => {
     });
     expect(handleParentClick).toHaveBeenCalledTimes(0);
   });
-
-  it('should have mouse enter simulated by test utils', async () => {
```

Comment:
Moved to `ReactTestUtils-test` since it was specific to test-utils. For synthetic event dispatches we have a test in https://github.com/facebook/react/blob/015ff2ed66c1d164111752263682d1d757c97f3e/packages/react-dom/src/__tests__/ReactDOMEventPropagation-test.js#L1292-L1363